security/operator/secopspolicy/apo_import.yaml

```diff
@@ -0,0 +1,18 @@
+# This collection of tasks will create an Aporeto
+# `networkaccesspolicies` policy.
+# Variables
+#   - pol_uid
+#   - k8s_uuid
+#   - apo_namespace
+#   - spec
+#   - template_name
+- name: Process j2 template
+  set_fact:
+    tdata: "{{ lookup('template', './templates/{{ template_name }}') }}"
+- name: Create policy
+  command: |
+    /usr/local/bin/apoctl api import \
+    --namespace {{ apo_namespace }} \
+    -f -
+  args:
+    stdin: "{{ tdata }}"
```

Comment:
Just a suggestion but you should be able to perform the lookup from line 11 at line 18 instead of using `set_fact` and a separate task.

Reply:
Good idea. I'll leave it for now and look into changing it in a a future change.